src/cascadia/TerminalSettingsModel/ColorScheme.h

```diff
@@ -55,12 +55,12 @@ namespace winrt::TerminalApp::implementation
     private:
         std::array<til::color, COLOR_TABLE_SIZE> _table;
 
-        friend class TerminalAppLocalTests::SettingsTests;
-        friend class TerminalAppLocalTests::ColorSchemeTests;
+        friend class SettingsModelLocalTests::SettingsTests;
+        friend class SettingsModelLocalTests::ColorSchemeTests;
     };
 }
 
-namespace winrt::TerminalApp::factory_implementation
+namespace winrt::Microsoft::Terminal::Settings::Model::factory_implementation
 {
     BASIC_FACTORY(ColorScheme);

```

Comment:
whoa you can construct a colorscheme out of the idl?

Reply:
Yeah. We don't need it right now, but we'll need it for Settings UI soon. Removed.